services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -99,7 +99,7 @@ async def get_node(request: web.Request) -> web.Response:
         raise web.HTTPNotFound(reason=f"Project {project_uuid} not found") from exc
 
 
-@routes.delete(f"/{VTAG}/projects/{{project_uuid}}/nodes/{{node_uuid}}")
+@routes.post(f"/{VTAG}/projects/{{project_uuid}}/nodes/{{node_uuid}}/retrieve")
```

Comment:
really? that's a good one...

Reply:
you can see @pcrespov replied with the reason below. I just noticed that last time I put this route in I copied without changing.